gematria/datasets/convert_bhive_to_llvm_exegesis_input.cc

```diff
@@ -332,6 +336,9 @@ int main(int argc, char* argv[]) {
       }
     }
 
+    if (report_progress)
+      std::cout << "Finished annotating block #" << file_counter << ".\n";
```

Comment:
I'd put all service messages (errors, warnings, but even progress) to stderr. stdout should be reserved for actual requested output of the program (we have none here, because we always write to a file).